pkg/knuu/knuu.go

```diff
@@ -193,16 +198,9 @@ func setDefaults(ctx context.Context, k *Knuu) error {
 		}
 	}
 
-	if k.MinioClient == nil {
-		k.MinioClient = &minio.Minio{
-			K8s: k.K8sClient,
-		}
-	}
-
```

Comment:
Building on comment, we would still remove this in order to make defining the `MinioClient` optional.